apps/server/.env.default

```diff
@@ -1,2 +1,2 @@
 MONGODB_URL=mongodb://localhost:27017/mongodb-container
-JWT_SECRET=bananas
```

Comment:
`JWT_SECRET=` needs to be here, no? you can delete `bananas` but we still need to know that a JWT_SECRET needs to exist in the `.env`